client/scss/components/_book-promo.scss

```diff
@@ -0,0 +1,53 @@
+// TODO: Colours still being worked on by Tomi - using 'smoke' temporarily
+.book-promo {
+  display: block;
+  text-decoration: none;
+
+  @include container-query(('m': ('6'))) {
+    margin-bottom: 0;
+  }
+
+  @include container-query(('l': ('6'), 'xl': ('6'))) {
+    background: color('smoke');
+    display: flex;
+    align-items: flex-start;
+    padding-bottom: (map-get($gutter-width, 'medium'));
+  }
+}
+
+.book-promo__image-container {
+  background: color('transparent');
+  .image {
+    width: 127px;
+
+    @include container-query(('l': ('6'), 'xl': ('6'))) {
+      width: 178px;
+      min-width: 178px;
+    }
+  }
+}
+
+.book-promo__text-container {
+  background: color('smoke');
+  margin-top: -(map-get($gutter-width, 'small'));
+  padding-top: (map-get($gutter-width, 'small'))*2;
+
+  @include container-query(('l': ('6'), 'xl': ('6'))) {
+    background: none;
+  }
+}
+
+.book-promo__item-number {
+  @include container-query(('l': ('6'), 'xl': ('6'))) {
+    display: none;
+  }
+}
+
+.book-promo__title {
+  transition: color 400ms ease;
+
+  .book-promo:hover &,
+  .book-promo:focus & {
+    color: color('smoke');
+  }
+}
```

Comment:
I might be being silly - but why is this all bespoke?
Could the grids not do what we needed?

Reply:
we use the grids for the layout of the promos as individual things, but the way the promo displays internally needs to be bespoke